lib/game.rb

```diff
@@ -72,4 +78,43 @@ def diagonal_down_sort
     end
     padded_matrix.transpose.map {|array| array.compact}
   end
+
+  def game_menu
```

Comment:
Lets ask in our project check in if this should go in the runner file or stay here since this class is getting a little chonky.